apps/sippy/deployment.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  labels:
-    app: sippy
   name: sippy
   namespace: sippy
+  labels:
+    app: sippy
```

Comment:
I think this ordering makes more sense to a human, but it will mean automated enforcement of yaml formatting/ordering can't be done with jq/yq.

I see the type is ordered this way in https://git.k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L116-L288 so presumably we could pass through `kubectl --dry-run` before something that enforced spacing/indenting standards

Reply:
Fair point. I'll admit I usually base things on how `helm create` generates their default yaml, but I'm totally fine with reverting the order if it helps our automation. 

```
$ cat testchart/templates/deployment.yaml | head -6
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{ include "testchart.fullname" . }}
  labels:
    {{- include "testchart.labels" . | nindent 4 }}
```

Let me know if I should re-order, and I can get another commit added today. 

Thanks!